<launch>

 <!-- Turtlesim Node-->
  <node pkg="donutdevice" type="donutdevice_node" name="donutdevice"/>


 <!-- PS3 joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="donut_joy" > 
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
  <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular" value="2" type="int"/>
  <param name="scale_linear" value="-20" type="double"/>
  <param name="scale_angular" value="20" type="double"/>
  <param name="scale_reverse" value="0.5" type="double"/>

  <node pkg="donutdevice" type="donut_teleop_ps3joy" name="teleop"/>
  <node respawn="true" pkg="rosserial_python" type="serial_node.py" name="rosserial">
    <param name="port" value="/dev/ttyACM0" />
    <param name="baud" value="57600" />
    <param name="cmd_vel" value="donutdevice/cmd_vel" />
  </node>

</launch>